packages/express-openapi/README.md

```diff
@@ -368,6 +368,57 @@ available in the affected operation handler method.
 * `'x-express-openapi-disable-validation-middleware': true` - Disables input
 validation middleware.
 
+#### Express Middleware to Validate Responses on All Routes
+
+By overriding the implementation of `res.send` within a middleware function, it is possible to perform `res.validateResponse` on all paths, and allow the paths to use `res.send` or `res.json`. The example below is how such a middleware might be added to `app.js`. For a TypeScript example see [TypeScript Example of Global Response Validation](#typescript-example-of-global-response-validation).
+
+```javascript
+function ValidateAllResponses(req, res, next) {
```

Comment:
can you lowercase this?  typically in javascript constructor functions are PascalCase.